File: WebApplication1/Models/Entities/Answer.cs
namespace WebApplication1.Models.Entities
{
    public class Answer
    {
        public int Id { get; set; }
        public required string Text { get; set; }
        public bool IsCorrect { get; set; } 

        //Foreign key
        public int QuestionId { get; set; }
        public Question? Question { get; set; }
    }
}

File: quizz_platform_flutter/lib/data/models/answer.dart
class Answer {
  final int? id;
  String text;
  bool isCorrect;

  Answer({
    this.id,
    required this.text,
    this.isCorrect = false,
  });

  // Tworzy obiekt z mapy (z API)
  factory Answer.fromJson(Map<String, dynamic> json) {
    return Answer(
      id: json['id'],
      text: json['text'] ?? '',
      isCorrect: json['isCorrect'] ?? false,
    );
  }

  // Zamienia obiekt na mapę (do API)
  Map<String, dynamic> toJson() {
    return {
      if (id != null) 'id': id,
      'text': text,
      'isCorrect': isCorrect,
    };
  }
}